venvUtils/ensureVenv.py

```diff
@@ -179,7 +179,7 @@ def ensureVenvAndRequirements():
 			"This means that transitive dependencies can get out of sync "
 			"with those used in automated builds. "
 			"Would you like to continue recreating the environment?",
-			default=True,
+			default=False,
```

Comment:
why is this changed?

Reply:
Ah, I only explained that in the commit summary.
When pre-commit runs something in the NVDA venv, it does so non-interactively. When requirements changed, the True value will effectively mean that pre-commit will recreate the venv. That is really overkill in many situations. Ive also encountered cases myself where I ran this non-interactively, causing my venv to be discarded.
I think it still makes sense to encourage people to recreate the venv especially if they don't know what they're doing, but non-interactive automation should impose the smallest possible impact on the venv IMO.